### PR TITLE
Clarify Houston API Upgrade Requirements When Moving from AC to Runtime

### LINKED ISSUE #1296: Request for graphQL version handling
**Request type:**

- [ ] **New document:**: Add the `docs-content` label to this issue.
- [ X] **Addition to an existing document**: Add the `enhancement` label to this issue.

**What is the purpose of the document/enhancement? Select all that apply:**

- [ ] Document a new feature for an upcoming release. Add the appropriate release version label to this issue.
- [ X] Information for an existing feature that isn't yet documented.
- [ X] Additional considerations/ reference material for an existing document.

**Problem/feature that requires documentation:**

Customers on Astronomer Certified will need to migrate to RunTime.  Those who leverage Houston graphQL for deployments will need to make changes to handle the AC version migrating to the RunTime version.  

This should be highlighted in:
* https://docs.astronomer.io/software/migrate-to-runtime 
Would be also helpful to mention in 
* https://docs.astronomer.io/software/houston-api#access-the-graphql-playground 

**Are you the right person to review a draft of the document for technical accuracy? If not, who is?**

No, possibly Constance?

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1296 

@cmarteepants the revised documentation is ready for your review. If you want examples of queries or mutations with the `airflowVersion` and `runtimeVersion` arguments, can you provide them, or direct me to someone who can? Thanks for your help to date.

### CODE REVIEW

**Comment by cmarteepants** [comment]
> Resolves #1296
> 
> @cmarteepants the revised documentation is ready for your review. If you want examples of queries or mutations with the `airflowVersion` and `runtimeVersion` arguments, can you provide them, or direct me to someone who can? Thanks for your help to date.

@aliotta could you please provide @kwcanuck with an example query or mutation that would need to be adjusted when moving to runtime? Perhaps the update image or deployment flow?

**Comment by aliotta** [comment]
> > Resolves #1296
> > @cmarteepants the revised documentation is ready for your review. If you want examples of queries or mutations with the `airflowVersion` and `runtimeVersion` arguments, can you provide them, or direct me to someone who can? Thanks for your help to date.
> 
> @aliotta could you please provide @kwcanuck with an example query or mutation that would need to be adjusted when moving to runtime? Perhaps the update image or deployment flow?

Here's the 2 most notable queries. createDeployment and updateDeploymentImage.

updateDeploymentImage(
releaseName: String!
image: String!
dockerconfigjson: JSON
airflowVersion: String
runtimeVersion: String
): Deployment

createDeployment(
workspaceUuid: Uuid!
releaseName: String
namespace: String
type: String!
label: String!
description: String
version: String
airflowVersion: String
runtimeVersion: String
executor: ExecutorType
workers: Workers
webserver: Webserver
scheduler: Scheduler
triggerer: Triggerer
dagDeployment: DagDeployment = {type: image}
config: JSON
properties: JSON
cloudRole: String
): Deployment

**Review by cmarteepants — APPROVED** [review]
This seems fine to me. Re example - it is really long, and I think the action items are clear as is. If we receive feedback from clients that they want an example, we can add something later.

File: software/houston-api.md [hunk 1]
```diff
@@ -18,27 +18,31 @@ For example, you can:
 - Add a user to a Workspace
 - Make a user a System Administrator
 
-Anything you can do via the Software UI, you can do programmatically via Astronomer's Houston API. Read below for guidelines.
+Anything you can do with the Software UI, you can do programmatically with the Astronomer Houston API.
+
+:::info
+
+  If you're using the Astronomer Houston API and you're migrating from Astronomer Certified (AC) to Astro Runtime, you'll need to replace `airflowVersion` arguments with `runtimeVersion` arguments in your scripts. For more information about migrating a Deployment from Astronomer Certified to Astro Runtime, see [Migrate a Deployment from Astronomer Certified to Astro Runtime](migrate-to-runtime.md).
```
Comment (jwitz) [inline software/houston-api.md:25]:
```suggestion
If you're using the Astronomer Houston API and you're migrating from Astronomer Certified (AC) to Astro Runtime, you'll need to replace `airflowVersion` arguments with `runtimeVersion` arguments in your scripts. For more information about migrating a Deployment from Astronomer Certified to Astro Runtime, see [Migrate a Deployment from Astronomer Certified to Astro Runtime](migrate-to-runtime.md).
```

File: software/houston-api.md [hunk 2]
```diff
@@ -18,27 +18,31 @@ For example, you can:
...
+
+  :::
```
Comment (jwitz) [inline software/houston-api.md:27]:
```suggestion
:::
```

**Review by jwitz — APPROVED** [review]

**Comment by kwcanuck** [comment]
Discussed with @cmarteepants and it was decided to merge this PR now. @rwalshe can create a new issue  for the additional changes when he returns from PTO.

**Pull request merged**